pkg/asset/manifests/cloudproviderconfig.go

```diff
@@ -92,6 +94,24 @@ func (cpc *CloudProviderConfig) Generate(dependencies asset.Parents) error {
 		}
 
 		cm.Data[cloudProviderConfigDataKey] = openstackmanifests.CloudProviderConfig(cloud.CloudConfig)
+
+		// Get the ca-cert-bundle key if there is a value for cacert in clouds.yaml
+		caFile := []byte{}
+		caFilePath := cloud.CloudConfig.CACertFile
+		if caFilePath != "" {
+			_, err := os.Stat(caFilePath)
+			if os.IsNotExist(err) {
+				return errors.Wrap(err, "cacert from clouds.yaml file path does not exist")
+			} else if err != nil {
+				return errors.Wrap(err, "could not read cacert from clouds.yaml")
+			}
+
+			caFile, err = ioutil.ReadFile(caFilePath)
+			if err != nil {
+				return errors.Wrap(err, "could not read cacert from clouds.yaml")
+			}
+		}
```

Comment:
Can you make this a reusable function? In a later commit, we'll also need to update the cert that is passed to terraform:

https://github.com/openshift/installer/blob/903d13c/pkg/asset/cluster/tfvars.go#L341